lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/MessagingTransportFactory.java

```diff
@@ -42,24 +41,12 @@
 
   private final String localAddress;
 
-  /**
-   * @deprecated Have an instance injected instead.
-   */
-  @Deprecated
   @Inject
   public MessagingTransportFactory(final LocalAddressProvider localAddressProvider) {
```

Comment:
In that case, this constructor should still be marked as deprecated. Noone should write new code calling it directly, correct?